docs/Usage/groups.md

```diff
@@ -44,4 +44,4 @@ Instead of assigning users "*one at a time*", we will assign the "Production" Pr
 
     <img src="/img/docs/groups-project-add.png" alt="Assigning groups to projects" />
 
-Read more about [features of Paralus](../usage/).
\ No newline at end of file
+Read more about [features of Paralus](../usage/).
```

Comment:
what's the change here?

Reply:
Don't think there is a change there, just a newline at the end.